### ISSUE TITLE
Claudia does not utilize local .npmrc file

### ISSUE THREAD

**Issue by tyrsius** [claudiajs/claudia#81]
* Expected behaviour:
`claudia create` deploys lambda function with a private registry.

* What actually happens:
```
packaging files	npm install --production
npm install --production failed.
```

the npm debug log shows
`'@nike/lambda-node-logger' is not in the npm registry.`

* Link to a minimal, executable project that demonstrates the problem:
I am unable to do this, because the private registry is not externally accessible.

* Steps to reproduce the problem:
use a local `.npmrc` file to link to a private registry
```
registry=http://artifactory.nike.com/artifactory/api/npm/npm-nike
@nike:registry=http://artifactory.nike.com/artifactory/api/npm/npm-nike/
```

`npm install --production` works from the CLI, but somehow claudia invokes it without the `.npmrc` file, which causes the above error.

**Comment by gojko** [comment]
is the .npmrc file in this case in your project directory or somewhere else?

**Comment by tyrsius** [comment]
It is in the project directory

**Comment by gojko** [comment]
OK, i'll fix that soon. Meanwhile, using .npmrc in your user home dir should allow you to work around this problem.

**Comment by gojko** [comment]
this is now available on NPM as version 2.1.3
